fastlane/spec/actions_specs/swiftlint_spec.rb

```diff
@@ -289,6 +289,56 @@
           expect(result).to eq("swiftlint lint")
         end
       end
+
+      context "when specify format option" do
+        it "adds format option" do
+          result = Fastlane::FastFile.new.parse("lane :test do
+            swiftlint(
+              mode: :autocorrect,
+              format: true
+            )
+          end").runner.execute(:test)
+
+          expect(result).to eq("swiftlint autocorrect --format")
+        end
+
+        it "omits the switch if mode is :lint" do
+          result = Fastlane::FastFile.new.parse("lane :test do
+            swiftlint(
+              mode: :lint,
+              format: true
+            )
+          end").runner.execute(:test)
+
+          expect(result).to eq("swiftlint lint")
+        end
+
+        it "omits the switch if swiftlint version is too low" do
```

Comment:
too earlier? (I'm not a native speaker)

Reply:
It's not right, but I basically copied and pasted. I'm gonna change this.